diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
index 4c506533866d411b18799e5441f02c2b3e9a0c44..598b08a7c700fb992c092b7aac090fd36f0983b2 100755
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -225,7 +225,6 @@ build_slice() {
     -destination "generic/platform=${destination}" \
     -derivedDataPath "$DERIVED_DATA_PATH" \
     -configuration "$CONFIGURATION" \
-    -quiet \
     -disableAutomaticPackageResolution \
     -skipPackagePluginValidation \
     -skipMacroValidation \
